packages/client/mutations/UpdatePokerScopeMutation.ts

```diff
@@ -12,6 +12,7 @@ import {
 graphql`
   fragment UpdatePokerScopeMutation_meeting on UpdatePokerScopeSuccess {
     meeting {
+      facilitatorStageId
```

Comment:
Am I right in saying that by including `facilitatorStageId` in this query, we let Relay know that it's being updated in the mutation and should be re-fetched? Even though we're not using it in the optimistic updater

Reply:
that is correct! for moving between stages I don't much bother with optimistic updates because sometimes we have to build the next stage (e.g. between voting & discuss).